ui/apos/components/AposDuplicateImportModal.vue

```diff
@@ -179,6 +179,13 @@ export default {
       return this.checked.length
         ? 'var(--a-white)'
         : 'transparent';
+    },
+    replaceDocIds() {
+      return this.checked.map(
+        aposDocId => this.duplicatedDocs.find(doc => doc.replaceId && doc.aposDocId === aposDocId)
+      )
+        .filter(Boolean)
+        .map(({ aposDocId, replaceId }) => [ aposDocId, replaceId ]);
```

Comment:
Maybe an object instead of an array would be clearer, and would avoid things like `replaceItem[1]` which is not really clear I think?

Reply:
this follows the backend argument requirements - see `await self.apos.doc.changeDocIds(replaceDocIdPairs, { skipReplace: true });` and the relevant code in `overrideDuplicates`.
For consistency, every "pair" is in that format.